src/Parsers/TatraBankaSimpleMailParser.php

```diff
@@ -14,80 +31,30 @@ public function parse($content)
     {
         $mailContent = new MailContent();
 
-        $cardpayPattern = '/VS=(.*) RES=(.*) AC=(.*) SIGN=(.*)/m';
-        $res = preg_match($cardpayPattern, $content, $result);
```

Comment:
Thank you for changing this!

Yesterday when i was reviewing your previous MR i was thinking about something like this ;-)